### PR TITLE
Implement `stdbool.h`

### PR DESCRIPTION
**Pull request by ribbanya**

### CODE REVIEW

**Review by ribbanya — COMMENTED** [review]
78f8d841cd69a0a2cbb80612e17a66dcc2c39d16 is just
```sh
replace_symbol BOOL bool
replace_symbol TRUE true
replace_symbol FALSE false
```

so please don't click through it.

File: src/MSL/stdbool.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/// @file
+/// @todo Assumes @c GEKKO.
+#ifndef STDBOOL_H
+#define STDBOOL_H
+
+/// A signed integer used to contain boolean values
+/// @todo Rename to @c bool.
```
Comment (r-burns) [inline src/MSL/stdbool.h:7]:
```suggestion
```

File: src/MSL/stdbool.h [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/// @file
+/// @todo Assumes @c GEKKO.
```
Comment (r-burns) [inline src/MSL/stdbool.h:2]: wdym assumes gekko?

Comment (ribbanya) [reply src/MSL/stdbool.h:2]: The C standard is an unsigned integer.

**Review by r-burns — APPROVED** [review]

**Pull request merged**